corehq/form_processor/models.py

```diff
@@ -940,6 +953,12 @@ def is_relevant(self):
         return relevant
 
     @property
+    def user_id(self):
+        if self.form:
+            return self.form.user_id
```

Comment:
yeah had a quick discussion when i first implemented and i wasn't sure if you had designed it like that to save space. looks like that wasn't the case. i agree i think it'd be good to store user_id on the transaction. will followup